modules/developer-cli-odo-installing-odo-on-linux-on-IBM-Power.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/installing-odo.adoc
+
+[id="installing-odo-on-linux-on-IBM-Power"]
+
+= Installing {odo-title} on Linux on IBM Power
+
+== Binary installation
+
+.Procedure
+
+. Obtain the binary:
++
+[source,terminal]
+----
+# curl -L https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-linux-ppc64le -o /usr/local/bin/odo
+----
+
+. Change the permissions on the file:
++
+[source,terminal]
+----
+# chmod +x /usr/local/bin/odo
+----
+
+== Tarball installation
```

Comment:
Please also ensure that the ID includes reference to the context ([id="module-name_{context}"]).

Reply:
As per the previous subsection, this subheaded section could potentially be split out into an individual module. This would help if another team wanted to reuse just this one section in future.

Reply:
@pneedle-rh this PR is from IBM writers who are only updating the docs for the IBM P and Z installation. I think it attempts to keep in sync with the existing docs. 
I think your suggestions would be relevant to this entire assembly: https://docs.openshift.com/container-platform/4.5/cli_reference/developer_cli_odo/installing-odo.html#tarball-installation 
The DevTools team works on these docs. I would suggest we park these improvements for now. We will create a Jira for this and take it up through another. Hope that works? 
@boczkowska wdyt?

Reply:
@boczkowska has given her ack, she is the core writer for odo. So we will track such changes in a jira and fix them via a separate PR.

Reply:
@Preeticp ok that makes sense. Please resolve the following items in this PR:

* Adding `_{context}` to the anchors
* Adding anchors to the subheadings
* Removing the merge conflict residue (e.g. `<<<<<<< HEAD`) so that the preview build as expected